File: pset6/mario/mario.py
from cs50 import get_int

while True:
  n = get_int("Height: ")
  if 1 <= n <= 8:
    break;

if (n == 1):
    print("#")
else:
  for i in range(2,n+1):
      print(" "*(n-i), "#"*(i-1))

  print("#"*i)

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
  int n;
    do
    {
      n = get_int("Height: ");
    }
    while (n < 1 || n > 8);
       for (int j = 1; j < n+1; j++)
       {
          for (int i = 0; i < n-j; i++)
                {
                  printf(" ");
                }
           for (int k = 0; k < j; k++)
                {
                  printf("#");
                }
            printf("\n");
        }
}